cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/servlet/UserTaskManagerTest.java

```diff
@@ -71,6 +71,22 @@ public void testCreateUserTask() {
     Assert.assertEquals(userTaskHeaderValue.getValue(), testUserTaskId.toString());
     Assert.assertEquals(future, future3);
 
+    EasyMock.reset(mockHttpServletResponse);
+    // test-case: for sync task, UserTaskManager does not create mapping between request URL and UUID.
+    HttpServletRequest mockHttpServletRequest3 = prepareRequest(null, null, "test_sync_request", Collections.emptyMap());
+    userTaskManager.getOrCreateUserTask(mockHttpServletRequest3, mockHttpServletResponse, uuid -> future, 0, false).get(0);
```

Comment:
Why do we not store and check whether the response is as expected -- i.e. is there nothing to verify?

Reply:
There isn't.